estimator/serenity_estimator.cpp

```diff
@@ -35,14 +35,22 @@ class SerenityEstimatorProcess :
 
   Future<Resources> _oversubscribable(
       const Future<ResourceUsage>& _resourceUsage) {
-    Try<Resources> ret = this->pipeline->run(_resourceUsage.get());
+    // TODO(bplotka): Decide if need such simple algorithm.
```

Comment:
I mimicked quickly the fixed  estimator here, so i didn't spent time to think about it. Maybe we want to make additional filter for that or do it in slack_observer? I was surprised that estimator is responsible for decreasing slack with allocated revocabled.. Why do we need to do that in estimator? We are decreasing the slack, then slave is including allocated revocables (https://github.com/apache/mesos/blob/master/src/slave/slave.cpp#L4251), then allocator is decresing again with same amount.. Are you able to elaborate on that? What is the reason from the mesos perspective?